pkg/controller/customdomain/customdomain_controller.go

```diff
@@ -227,6 +228,11 @@ func (r *ReconcileCustomDomain) Reconcile(request reconcile.Request) (reconcile.
 	}
 	reqLogger.Info(fmt.Sprintf("The ingresscontroller %s already exists in the %s namespace", ingressName, ingressOperatorNamespace))
 
+	// Set domain in status in wildcard form
+	if len(instance.Status.Domain) == 0 {
```

Comment:
These would be `nil` no?

Consideration, should we store these and `.Status.Endpoint` in the `.Spec` instead? This would remove the need to restore status to get state. Seems like these wouldn't change once set.

Reply:
I've removed `Domain` from the status struct and now using the .Spec.Domain for the table column.